lib/helpers/UiBaseLocators.ts

```diff
@@ -159,13 +162,14 @@ export class UiBaseLocators {
     this.addPropertyBtn = page.getByLabel('Add property', {exact: true});
     this.typeToFilterSearchTxt = page.locator('[type="search"] #input');
     this.editorSettingsBtn = page.getByLabel('Editor settings');
-    this.labelOnTopBtn = page.getByRole('button', {name: 'Label on top'});
+    this.labelAboveBtn = page.locator('button').filter({hasText: 'Label above (full-width)Label'});
```

Comment:
It would be better if you can locate this element in another way since the text seems be changed.